test-data/packages/typedpkg_c_ext/setup.py

```diff
@@ -0,0 +1,12 @@
+from setuptools import setup, Extension
+
+setup(
+    name='typedpkg_c_ext',
+    version='1.0',
+    packages=['typedpkg_c_ext'],
+    ext_modules=[Extension('typedpkg_c_ext.hello', ['hello.c'])],
```

Comment:
Looks like you can just delete this line (and the file hello.c) and the test works just as well. (Verified by removing the `isdir()` check from modulefinder.c line 124.)